src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -134,6 +161,22 @@ public void DeleteUpdatableWorkloadsFile()
             }
         }
 
+        public static string WorkloadSetVersionToWorkloadSetPackageVersion(string setVersion)
+        {
+            var nugetVersion = new NuGetVersion(setVersion);
+            var patch = nugetVersion.Revision;
+            var release = string.IsNullOrWhiteSpace(nugetVersion.Release) ? string.Empty : $"-{nugetVersion.Release}";
+            return $"{nugetVersion.Major}.{nugetVersion.Patch}.{patch}{release}";
+        }
+
+        public static string WorkloadSetPackageVersionToWorkloadSetVersion(SdkFeatureBand sdkFeatureBand, string packageVersion)
```

Comment:
It looks like this wasn't addressed yet, it would be good to sync the implementations with arcade, especially since there are tests covering arcade's version of this.